cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -737,55 +737,106 @@
   <!-- Kafka Server Configuration -->
 
   <property>
-    <name>kafka.bind.address</name>
```

Comment:
Since this was a public config as well, I think you need to deprecate the `kafka.bind.address` with the `kafka.server.host.name`.